### ISSUE TITLE
Notification button padding

### ISSUE THREAD

**Issue by AccountOneOff** [julio641742/extend-panel-menu#29]
The notification button has too much padding. Version 5.3

![Notifcation padding](https://i.imgur.com/TPOGcZf.png)

**Comment by julio641742** [comment]
Working on it

**Comment by julio641742** [comment]
Could you check the latest commit and see if you still have the same problem

**Comment by AccountOneOff** [comment]
Just updated and the problem still persists.

**Comment by julio641742** [comment]
Could you check again with the latest commit :+1:

**Comment by AccountOneOff** [comment]
I found the problem. For some reason when running the makefile the icons folder doesn't transfer over to the extension folder so no indicator appears. I had to set [this line](https://github.com/julio641742/extend-panel-menu/blob/master/indicators/notification.js#L43) to `true` for it to be visible which uses my icon themes notification icon and causes it to scale poorly. I set it back to `false` and I pasted the icons folder into the extension folder and everything is fine now. :smiley:
